src/components/animations/CheckmarkAnimation.js

```diff
@@ -1,83 +1,106 @@
 import React from 'react';
-import { View } from 'react-native';
 import RadialGradient from 'react-native-radial-gradient';
 import Animated, {
-  Easing,
-  useAnimatedStyle,
-  useDerivedValue,
-  withSequence,
+  withDelay,
+  withSpring,
   withTiming,
 } from 'react-native-reanimated';
-import { Path, Svg } from 'react-native-svg';
+import { LargeCheckmarkIcon } from '../icons/svg/LargeCheckmarkIcon';
 import { Flex } from '../layout';
 import styled from '@rainbow-me/styled-components';
 
 const Container = styled(Flex)({
```

Comment:
Sorry, I didn't see this `CheckmarkAnimation` component in the previous reviews.

Ideally we should be using TypeScript for new components, would you be able to convert this file to `.tsx`?

Also, can we use the design system instead of styled-components?

This should mimic the `Container` styles:

```tsx
<Box alignItems="center" height={{ custom: 132 }} justifyContent="center" width="full">
```

Note: adhoc/once off sizes that don't confine to the typical system sizes (such as 132px) are defined via a `custom` attribute on `height`.